src/custom/hooks/useSwapCallback.ts

```diff
@@ -109,11 +109,11 @@ async function postOrder(params: PostOrderParams): Promise<string> {
     order: {
       id: orderId,
       owner: account,
-      ...unsignedOrder,
       creationTime,
       signature,
       status: OrderStatus.PENDING,
-      summary
+      summary,
+      ...unsignedOrder
```

Comment:
think it's safe here but should destructure at top as this could override previous props if any are similar

```js
let b = { 
  c: 3, 
  d: 4, 
  a: 5, 
  b: 6  
}
let c = {
    a: 1,
    b: 2,
    ...b
}

// { a: 5, b: 6, c: 3, d: 4 } c.a and c.b are overwritten by b.a and b.b
```

Reply:
okay, I thought it would be ok since we always want to inlcude all the fields of this `unsignedOrder`. In fact, is used in the order type definition. 

I don't have strong opinion, so I'll go for it

Reply:
well you can, just destructure unsigned order first, not last as you're doing.